tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_empty_fields.py

```diff
@@ -0,0 +1,171 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh is able to detect vulnerabilities in the applications installed in agents using the Vulnerability Detector
+       module. This software audit is performed through the integration of vulnerability feeds indexed by Redhat,
+       Canonical, Debian, Amazon Linux and NVD Database.
+
+components:
+    - vulnerability_detector
+
+suite: feeds
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-modulesd
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/index.html
+
+tags:
+    - vulnerability_detector
+    - cpe_helper
+'''
+import os
```

Comment:
Remove this. Use Path:
```suggestion
from pathlib import Path
```
Apply this in all tests.

Reply:
Done in https://github.com/wazuh/wazuh/pull/17576/commits/cf428669ca6378ddac6a6dd7c45865851a1b1194